cpp/test/IceSSL/configuration/OpenSSLTests.cpp

```diff
@@ -317,12 +391,76 @@ serverValidatesClientUsingCAFile(Test::TestHelper* helper, const string& testDir
             { SSL_set_verify(ssl, SSL_VERIFY_PEER | SSL_VERIFY_FAIL_IF_NO_PEER_CERT, nullptr); }};
         Ice::CommunicatorHolder serverCommunicator(createServer(serverAuthenticationOptions, helper));
 
-        Ice::CommunicatorHolder clientCommunicator(createClient(Ice::SSL::ClientAuthenticationOptions{
+        Ice::SSL::ClientAuthenticationOptions clientAuthenticationOptions{
```

Comment:
what's the change here (for `serverValidatesClientUsingCAFile`) fixing?

Reply:
Is still the same test I move the creation of the options out of the method call.